src/qibolab/instruments/emulator/readout.py

```diff
@@ -99,8 +109,8 @@ def simulate_excited_state_iq(self, pulse: ReadoutPulse):
         return self.simulate_and_demodulate(s21, pulse)
 
     def simulate_and_demodulate(self, s21: complex, pulse: ReadoutPulse):
-        """Simulates the readout pulse for a given S21-parameter and
-        demodulates it.
+        """Simulates the readout pulse for a given S21-parameter and homodyne
+        demodulation/2nd stage of heterodyne demodulation.
```

Comment:
You may want to provide an inclusion for a LO frequency so that the IF frequency is parsed at the sampling rate instead of the full pulse frequency.

Reply:
sure, i have added them